server/apiCalls/ladokApi.js

```diff
@@ -1,17 +1,31 @@
 'use strict'
 
 const { createApiClient } = require('om-kursen-ladok-client')
-const serverConfig = require('../configuration').server
+const { server: serverConfig } = require('../configuration')
+
+const client = createApiClient(serverConfig.ladokMellanlagerApi)
 
 async function getCourseAndActiveRounds(courseCode, language) {
-  const client = createApiClient(serverConfig.ladokMellanlagerApi)
   const [course, rounds] = await Promise.all([
     client.getLatestCourseVersion(courseCode, language),
     client.getActiveCourseRounds(courseCode, language),
   ])
   return { course, rounds }
 }
 
+async function getExaminationModules(utbildningstillfalleUid, language) {
+  try {
+    const examinationModules = await client.getExaminationModulesByUtbildningstillfalleUid(
```

Comment:
This  function (getExaminationModulesByUtbildningstillfalleUid) that you created for kurs-pm-data-admin-web will uses a tillfälles uid to look up the correct utbildningsinstans uid and then fetch moduler. 

But here, in kursinfo-web, we already have the course (utbildningsinstans). So I think you should add uid in mapCourse and the create a new getExaminationModulesBy*UtbildningsinstansUid* and just pass the already fetch uid to avoid extra calls.